ui/summary.go

```diff
@@ -277,7 +286,13 @@ func SummarizeMetrics(w io.Writer, indent string, t time.Duration, timeUnit stri
 		if sink, ok := m.Sink.(*stats.TrendSink); ok {
 			cols := make([]string, len(TrendColumns))
 			for i, col := range TrendColumns {
-				value := m.HumanizeValue(col.Get(sink), timeUnit)
+				var value string
```

Comment:
I would write like this:
```
value := m.HumanizeValue(col.Get(sink), timeUnit)
if col.Key == "count" {
    value = strconv.FormatFloat(col.Get(sink), 'f', 0, 64)
}
```

Reply:
Hhmm I agree that that's cleaner, but this way would essentially process the `count` value twice, when there's no reason to call `m.HumanizeValue()` on it. Negligible performance impact, to be sure, but it feels better to me to not execute code needlessly.

Reply:
I think this is fine and you can leave as it is @imiric